### DISCUSSION TITLE
Unavailable in USD

### DISCUSSION THREAD

**Discussion by n8yeung** [swellstores/community#96] [Help] [answered]
![image](https://user-images.githubusercontent.com/5014417/213901386-29578c8c-1aea-40a9-bb2a-0a3e99c7b141.png)
Why is this happening?

**Comment by FeMaffezzolli** [comment]
The same for me. The weird thing is that if I refresh the page (product page) then it works fine. I’m using the Brazilian location only.

**Comment by jesseswell** [comment]
Hey @n8yeung!

There are a couple of possible reasons here:

1. It could be possible that you change your store currency, and you're viewing a demo product (the demo product is in USD by default ).
2. You could have changed your currency in your store setting but haven't republished your storefront.
3. You could be using an outdated version of the Origin theme.
4. Possibly a cache-related issue. 

Are you using a custom storefront? If yes, It could be item 4.

**Reply by FeMaffezzolli** [reply, marked as answer]
Hey @jesseswell, even though the answer was not for I'll respond anyway since I'm facing the same problem.

1. It could be possible that you change your store currency, and you're viewing a demo product (the demo product is in USD by default ).

From the admin panel, how can I tell a product is demo or not? I couldn't find this flag. In my case, I removed all the products that already existed when created the store.

And how exactly can I change the store currency? In the storefront editor there is a switch to choose if I want or not the display this selector to the user. Event thought I define true, I can't see anything on the page.

![image](https://user-images.githubusercontent.com/26548010/214048754-d3d26e4f-0950-4707-8628-d8ad5d51f6b6.png)

2. You could have changed your currency in your store setting but haven't republished your storefront.

I have a feeling that this might be the problem. But I opened another issue this morning since I'm not able to publish new version of the storefront from the editor. I can see the changes reflected on dev env, but not on prod. I think there's some cache thing going on. https://github.com/swellstores/community/issues/104

3. You could be using an outdated version of the Origin theme.

The version I see on the editor is v2.0.10

4. Possibly a cache-related issue.

Yes, seems to be the case.

**Comment by steven-k1** [comment]
@n8yeung Would you be able to confirm if you have configured your currency settings with a fallback to USD?

I can also check on our end if you are able to provide the account ID that you are seeing this issue with.
